config/source/100-deprecated-namespace.yaml

```diff
@@ -1,20 +0,0 @@
-# Copyright 2019 The Knative Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: knative-sources
```

Comment:
we do need this, b/c of:
https://github.com/knative-sandbox/eventing-kafka/blob/master/config/source/500-deprecated-controller.yaml#L15-L19

AFTER the release is done, we will remove the two deprecated files